esp/src/src-react/components/MetricsOptions.tsx

```diff
@@ -67,16 +67,18 @@ const GridOptions: React.FunctionComponent<GridOptionsProps> = ({
         }
     }, [selectionHandler, strSelection]);
 
-    return <FluentGrid
-        data={data}
-        primaryID={"id"}
-        columns={columns}
-        selectionMode={SelectionMode.multiple}
-        setSelection={selectionHandler}
-        setTotal={setTotal}
-        refresh={refreshTable}
-        height={`${innerHeight}px`}
-    ></FluentGrid>;
+    return <div style={{ position: "relative", height: 400 }}>
```

Comment:
Would height `100%` work here?

Reply:
No, not sure why, tbh. 100% height was what I'd added in a couple of other places, for some WU or files tabs, but it didn't in this case.